ngraph/core/include/ngraph/op/util/multi_subgraph_base.hpp

```diff
@@ -0,0 +1,232 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/op/parameter.hpp>
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace util
+        {
+            /// \brief Abstract base class for sub-graph based ops, i.e ops that have sub-graph
+            ///
+            class NGRAPH_API MultiSubGraphOp : public Op
```

Comment:
@itikhono do we have a time to reuse one `SubGraph` class before the 2021.4 release?